pkgs/tools/package-management/fortran-fpm/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, pkgs, fetchurl }:
+
+let
+  pname = "fortran-fpm";
+
+  version = "0.9.0";
+
+  src = fetchurl {
+    url = "https://github.com/fortran-lang/fpm/releases/download/v${version}/fpm-${version}.F90";
+    sha256 = "sha256-VWs4g7odtv1iyZunFD8el+u0CXKcQgnwOqPG/JcMzj8=";
+  };
+
+  buildPath = "build/bootstrap";
+in
+stdenv.mkDerivation {
+  pname = pname;
+  version = version;
+
+  src = src;
+  dontUnpack = true;
+
+  nativeBuildInputs = [ pkgs.gfortran ];
```

Comment:
```suggestion
  nativeBuildInputs = with pkgs; [ 
    gfortran
  ];
```
formatting nitpick

Reply:
I moved pkgs out to the `with` clause but left this as a single line